rust/routee-compass-core/src/model/traversal/default/grade/grade_configuration.rs

```diff
@@ -5,10 +5,7 @@ use serde::{Deserialize, Serialize};
 #[derive(Serialize, Deserialize, Clone, Debug)]
 pub struct GradeConfiguration {
     /// file with dense mapping from edge id to grade value
-    /// note: this is optional as a stop-gap to allow for loading energy models that do not require grade.
-    /// this is because we currently cannot define input features that are _optional_, and so
-    /// if a dummy grade model were not provided, energy models would fail to build.
-    pub grade_input_file: Option<String>,
+    pub grade_input_file: String,
```

Comment:
> I also made the grade model input required since it would prevent this silent error in the future

this was a dumb stopgap, i think i agree it's not needed.